tests/Fixture/PanelsFixture.php

```diff
@@ -42,7 +43,7 @@ class PanelsFixture extends TestFixture
         'title' => ['type' => 'string'],
         'element' => ['type' => 'string'],
         'summary' => ['type' => 'string'],
-        'content' => ['type' => 'binary'],
+        'content' => ['type' => 'binary', 'length' => TableSchema::LENGTH_LONG],
```

Comment:
Thanks for finding in which version it was added. I will update the minimum version constraints.